src/main/java/uk/gov/hmcts/darts/common/entity/SecurityGroupEntity.java

```diff
@@ -33,41 +41,52 @@ public class SecurityGroupEntity extends CreatedModifiedBaseEntity {
     @SequenceGenerator(name = "grp_gen", sequenceName = "grp_seq", allocationSize = 1)
     private Integer id;
 
+    @NotAudited
```

Comment:
This should probably be audited - when we create a new security group it would be valuable to know which role was assigned to it (could be either TRANSCRIBER or TRANSLATION_QA).

Reply:
Disregard - role cannot be changed so there is no point in auditing it.